oecophylla/cli/tests/data/envs.yml

```diff
@@ -0,0 +1,10 @@
+anvi: source activate oecophylla-anvi
+humann2: source activate oecophylla-humann2
+kraken: source activate oecophylla-kraken
+qc: source activate oecophylla-qc
+raw: source activate oecophylla-qc
```

Comment:
both qc and raw use the same environment?

Reply:
good question: @tanaes what do you think?

Reply:
We currently actually have separate environments available for raw and qc

Reply:
But see issue #56 ... probably good to combine them anyway.